source/features/hide-user-forks.tsx

```diff
@@ -7,16 +7,34 @@ function addSourceTypeToLink(link: HTMLAnchorElement): void {
 	link.search = String(search);
 }
 
+const skipUrlsWithType = ':not([href*="&type="])';
+
+const selectors = [
+	// User repos
+	`a[href$="?tab=repositories"]:is([href^="/"], [href^="${location.origin}/"])${skipUrlsWithType}`,
+
+	// Organization repos
+	`a[href$="/repositories"]:is([href^="/orgs/"], [href^="${location.origin}/orgs/"])${skipUrlsWithType}`,
```

Comment:
Same here, `skipUrlsWithType` won't match. Given the `/orgs` prefix I think it's safe to do this:

```suggestion
	`a[href*="/repositories"]:is([href^="/orgs/"], [href^="${location.origin}/orgs/"])${skipUrlsWithType}`,
```